### PR TITLE
Feat: account verification

### LINKED ISSUE #3: Profile verification
## Feature request

- [X] **I have looked at all the other requests before I submitted this issue**

### Summary

Create a button in the profile view where users cand send their own profile to verification

### Suggested solution(s)

Add a button 'Verify' in the profile section with [PrimeVue Buttons](https://www.primefaces.org/primevue/button), it'll be great if you use the **help** severity. When a user presses the button, it appears a [PrimeVue ConfirmDialog](https://www.primefaces.org/primevue/confirmdialog) to confirm (obviously) or deny the action. Finally, if the user confirmed or denied the verification, there is a [PrimeVue Toast](https://www.primefaces.org/primevue/toast) according to what the action was (success severity if the user confirmed or error severity if the user denied).

### PR DESCRIPTION
**Pull request by amenes12**
### What does it do

Implements the user interface to start an account verification process

### Usage

Big users might want to have some extra benefits using WAW, verifyng their accounts is the way to that!

### Why is it needed

User need to have a safe app full of other trustworthy users

### Related issue(s)/PR(s)

Fixes #3

### CODE REVIEW

File: src/core/components/base-header.vue [hunk 1]
```diff
@@ -14,6 +17,37 @@ const router = useRouter();
...
 const user = ref(auth.user);
 
+const confirm = useConfirm();
+const toast = useToast();
+
+const handleConfirmation = () => {
+  confirm.require({
+    header: "Account verification",
+    message:
+      "Do you want your account to be verified by WAW? These action might take weeks and it's irreversible.",
```
Comment (dalbitresb12) [inline src/core/components/base-header.vue]: Could you make this message less like a dangerous action and more like an informative message about being verified?

File: src/core/components/base-header.vue [hunk 2]
```diff
@@ -14,6 +17,37 @@ const router = useRouter();
...
+        severity: "success",
+        summary: "Verification requested",
+        detail:
+          "You are now under verification proccess. We'll get back to you by email soon!",
+        life: 3000,
+      });
+    },
+    reject: () => {
+      toast.add({
+        severity: "warn",
```
Comment (dalbitresb12) [inline src/core/components/base-header.vue]: I think this just should be an info message, a warning is a little bit too much.

**Review by dalbitresb12 — CHANGES_REQUESTED** [review]
Also, at least in staging there's a bug that shows the dialog twice.

image

File: src/core/components/base-header.vue (hunk 1 above)
Comment (amenes12) [reply src/core/components/base-header.vue]: Uhmmmmm, that's odd. The double dialog problem doesn't happen in dev... 

Also yes, the messages will be friendlier

File: src/core/components/base-header.vue (hunk 2 above)
Comment (amenes12) [reply src/core/components/base-header.vue]: Alright, got it!

**Comment by amenes12** [comment]
[Implementar el sistema de verificación de perfiles](https://trello.com/c/vUe2LZAJ/2-implementar-el-sistema-de-verificaci%C3%B3n-de-perfiles)

**Review by dalbitresb12 — APPROVED** [review]
Looks good to me!

**Pull request merged**